test/functional/http/post/4.multisignature.advanced.js

```diff
@@ -402,4 +402,17 @@ describe('POST /api/transactions (type 4) register multisignature', () => {
 			pendingMultisignatures
 		);
 	});
+
+	describe('verification', () => {
+		it('sending already confirmed multi-signature should fail', () => {
```

Comment:
conbsistency - `multisignature`, but what multisignature? registration, spend from multisignature account, etc? Please be precise.